sw/device/silicon_creator/rom_ext/doc/rescue.md

```diff
@@ -0,0 +1,143 @@
+# ROM_EXT Rescue Protocol
```

Comment:
Can you add a cross-reference to SUMMARY.md to make this document discoverable in the website?

Reply:
Done.